writeopia/writeopia.podspec

```diff
@@ -35,5 +35,5 @@ Pod::Spec.new do |spec|
             SCRIPT
         }
     ]
-    spec.resources = ['build/compose/ios/common/compose-resources']
```

Comment:
Looks like this is being changed because you use Windows. I don't understand much about iOS... so I don't know if this file should be in .gitignore or not. But for now, let's not change it, what do you think?

Reply:
Of course, I saw this commit and thought I hadn't committed this file, but here we are

Reply:
Fixed